muswarmlogger/loggers/sparql.py

```diff
@@ -245,16 +245,15 @@
     container = (await event.container) if event.status == "start" else None
     e2rdf = Event2RDF()
     e2rdf.add_event_to_graph(event.data, container=container)
-    ntriples = e2rdf.serialize().decode("utf-8")
-    await sparql.update("""
-        WITH <%(graph)s>
+    triples = e2rdf.serialize()
+    await sparql.update(
+        """
         INSERT DATA {
-            %(ntriples)s
+            GRAPH {{graph}} {
+                {{}}
+            }
```

Comment:
`WITH <...> INSERT DATA` does not exist in standard SPARQL. It's `INSERT DATA { GRAPH <...>`